project/Dependencies.scala

```diff
@@ -0,0 +1,24 @@
+import sbt._
+
+object Dependencies {
+  val playJson24: String = "2.4.3"
+  val playJson27: String = "2.7.4"
+
+  def playWsJson(scalaVersion: String): Seq[ModuleID] = {
+    val playJsonVersion = CrossVersion.partialVersion(scalaVersion) match {
+      case _ if sys.props.contains("playVersion") => sys.props.get("playVersion").get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5238576)